.github/workflows/test_conda-build.yml

```
@@ -1,9 +1,10 @@
-name: conda builds
+name: conda build
```

I would be tempted to leave as is, because renaming it will cause it to show up as a new workflow in the repo actions view (which is already quite busy)
```suggestion
name: conda builds
```

---

Fair, you can remove workflows from GH actions tab by removing previous runs.